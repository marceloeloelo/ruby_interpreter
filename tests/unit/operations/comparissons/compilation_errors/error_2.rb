true > nil # NoMethodError: undefined method `>' for true:Bool
