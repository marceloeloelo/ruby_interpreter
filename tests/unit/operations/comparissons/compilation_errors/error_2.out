NoMethodError: undefined method `>' for true:Bool
